buildroot-external/board/hardkernel/odroid-c4/meta

```diff
@@ -0,0 +1,12 @@
+# needs to match something in https://version.home-assistant.io/stable.json
+BOARD_ID=odroid-n2
```

Comment:
This should be `odroid-c4` as well

Reply:
Right; I was hoping we could get the C4 to appear on https://version.home-assistant.io/stable.json (whatever it takes to do that), and then I can update these.

Reply:
This should now be
```suggestion
BOARD_ID=odroid-c4
```